.idea/.gitignore

```diff
@@ -0,0 +1,8 @@
+# Default ignored files
```

Comment:
These are IDE auto generated files - we should remove this and the one below

Reply:
sure, I updated the changes to reflect this